roles/epel_repositories/tasks/main.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: 'Setup Extra Common if Stream'
+  yum:
+    name: "centos-stream-repos"
+    state: latest
+  tags:
+    - packages
+  when: ansible_distribution_major_version == '9'
```

Comment:
This needs an "os == centos" as otherwise that wont exist

Reply:
Adding `ansible_distribution == 'CentOS'`